.github/workflows/build-test-package.yml

```diff
@@ -3,279 +3,14 @@ name: Build, test, package
-        include:
-          - itk-python-git-tag: "v5.3rc04"
-
-    steps:
-    - name: Get specific version of CMake, Ninja
-      uses: lukka/get-cmake@v3.21.1
-
-    - uses: actions/checkout@v2
-      with:
-        path: "im"
-
-    - name: 'Install Python'
-      run: |
-        $pythonArch = "64"
-        $pythonVersion = "3.${{ matrix.python-version-minor }}"
-        iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/scikit-build/scikit-ci-addons/master/windows/install-python.ps1'))
-
-    - name: 'Fetch build dependencies'
-      shell: bash
-      run: |
-        mv im ../../
-        cd ../../im
-        curl -L "https://github.com/InsightSoftwareConsortium/ITKPythonBuilds/releases/download/${{ matrix.itk-python-git-tag }}/ITKPythonBuilds-windows.zip" -o "ITKPythonBuilds-windows.zip"
-        7z x ITKPythonBuilds-windows.zip -o/c/P -aoa -r
-        curl -L "https://data.kitware.com/api/v1/file/5c0ad59d8d777f2179dd3e9c/download" -o "doxygen-1.8.11.windows.bin.zip"
-        7z x doxygen-1.8.11.windows.bin.zip -o/c/P/doxygen -aoa -r
-        curl -L "https://data.kitware.com/api/v1/file/5bbf87ba8d777f06b91f27d6/download/grep-win.zip" -o "grep-win.zip"
-        7z x grep-win.zip -o/c/P/grep -aoa -r
-
-    - name: 'Build 🐍 Python 📦 package'
-      shell: cmd
-      run: |
-        cd ../../im
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
-        set PATH=C:\P\grep;%PATH%
-        set CC=cl.exe
-        set CXX=cl.exe
-        C:\Python3${{ matrix.python-version-minor }}-x64\python.exe C:\P\IPP\scripts\windows_build_module_wheels.py --py-envs "3${{ matrix.python-version-minor }}-x64" --no-cleanup
-
-    - name: Publish Python package as GitHub Artifact
-      uses: actions/upload-artifact@v1
-      with:
-        name: WindowsWheel3.${{ matrix.python-version-minor }}
-        path: ../../im/dist
-
-  publish-python-packages-to-pypi:
-    needs:
-      - build-linux-python-packages
-      - build-macos-python-packages
-      - build-windows-python-packages
-    runs-on: ubuntu-20.04
-
-    steps:
-    - name: Download Python Packages
-      uses: actions/download-artifact@v2
-
-    - name: Prepare packages for upload
-      run: |
-        ls -R
-        for d in */; do
-          mv ${d}/*.whl .
-        done
-        mkdir dist
-        mv *.whl dist/
-        ls dist
-
-    - name: Publish 🐍 Python 📦 package to PyPI
-      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
-      uses: pypa/gh-action-pypi-publish@master
-      with:
-        user: __token__
-        password: ${{ secrets.pypi_password }}
+  cxx-build-workflow:
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@246883ef3828fc00219145aeec9efa67b9889d0b
+    with:
+      itk-cmake-options: '-DITK_BUILD_DEFAULT_MODULES:BOOL=OFF -DITKGroup_Core:BOOL=ON'
```

Comment:
```suggestion
      itk-cmake-options: '-DITK_BUILD_DEFAULT_MODULES:BOOL=OFF -DITKGroup_Core:BOOL=ON -DModule_ITKAntiAlias:BOOL=ON -DModule_ITKImageGrid:BOOL=ON -DModule_ITKSpatialObjects:BOOL=ON -DModule_ITKTestKernel:BOOL=ON -DModule_ITKMetaIO:BOOL=ON'
```

Reply:
Add dependencies as per https://github.com/InsightSoftwareConsortium/ITKGrowCut/blob/master/itk-module.cmake

Reply:
Great! Thanks for taking the lead on this.